<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_sort">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>25</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Sorting</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>785</width>
      <height>222</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <row>
     <property name="text">
      <string>Selection Sort</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Shell Sort</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Quick Sort</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Merge Sort</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Counting Sort</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1024</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4096</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>16384</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>65536</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>262144</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>1048576</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4194304</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="1" column="6">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="2" column="6">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="5">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="3" column="6">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="5">
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item row="4" column="6">
     <property name="text">
      <string>-</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_chart">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>150</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Charts</string>
    </property>
   </widget>
   <widget class="QChartView" name="widget_lineChart" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>661</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="widget_pieChart" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>501</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
